packages/wonder-blocks-dropdown/src/components/__tests__/dropdown-core.test.js

```diff
@@ -797,4 +731,102 @@ describe("DropdownCore", () => {
+                    testId={`item-${i}`}
+                />
+            ),
+            focusable: true,
+            populatedProps: {},
+        }));
+
+        it("should render a virtualized list of options and focus on the search field", async () => {
+            // Arrange
+            render(
+                <DropdownCore
+                    initialFocusedIndex={undefined}
+                    onSearchTextChanged={jest.fn()}
+                    searchText=""
+                    // mock the items
+                    items={[
+                        {
+                            component: (
+                                <SearchTextInput
+                                    testId="search-text-input"
+                                    key="search-text-input"
+                                    onChange={jest.fn()}
+                                    searchText={""}
+                                />
+                            ),
+                            focusable: true,
+                            populatedProps: {},
+                        },
+                        ...optionItems,
+                    ]}
+                    role="listbox"
+                    open={true}
+                    // mock the opener elements
+                    opener={<button />}
+                    openerElement={null}
+                    onOpenChanged={jest.fn()}
+                />,
+            );
+
+            await screen.findByRole("listbox");
+
+            // Act
+            const searchField = await screen.findByPlaceholderText("Filter");
+
+            // Assert
+            expect(searchField).toHaveFocus();
+        });
+
+        it("should focus on the item after clicking on it", async () => {
+            // Arrange
+            render(
+                <DropdownCore
+                    initialFocusedIndex={undefined}
+                    onSearchTextChanged={jest.fn()}
+                    searchText=""
+                    // mock the items
+                    items={[
+                        {
+                            component: (
+                                <SearchTextInput
+                                    testId="search-text-input"
+                                    key="search-text-input"
+                                    onChange={jest.fn()}
+                                    searchText={""}
+                                />
+                            ),
+                            focusable: true,
+                            populatedProps: {},
+                        },
+                        ...optionItems,
+                    ]}
+                    role="listbox"
+                    open={true}
+                    // mock the opener elements
+                    opener={<button />}
+                    openerElement={null}
+                    onOpenChanged={jest.fn()}
+                />,
+            );
+
+            await screen.findByRole("listbox");
```

Comment:
Do we need to wait for this role, it seems like it should be available right away.